pkg/tfvars/openstack/openstack.go

```diff
@@ -95,6 +96,23 @@ func TFVars(
 		workermpool = installConfig.Config.Compute[0].Platform.OpenStack
 	}
 
+	var userManagedLoadBalancer bool
+	if installConfig.Config.Platform.OpenStack.LoadBalancer == nil {
+		userManagedLoadBalancer = false
+	} else {
+		loadBalancerType := installConfig.Config.Platform.OpenStack.LoadBalancer.Type
+		switch loadBalancerType {
+		case configv1.LoadBalancerTypeUserManaged:
+			userManagedLoadBalancer = true
+		case configv1.LoadBalancerTypeOpenShiftManagedDefault:
+			userManagedLoadBalancer = false
+		case "":
```

Comment:
How is empty a valid value? We should have validated install-config at this point, and set a valid default too.

That way, you only need to check for `installConfig.Config.Platform.OpenStack.LoadBalancer.Type == configv1.LoadBalancerTypeUserManaged` to get `userManagedLoadBalancer`.

Reply:
Right. I'll remove the empty case. Good point.